scripts/bootstrap-aio.yml

```diff
@@ -223,6 +223,30 @@
                     rbd_secret_uuid: "{%raw%}{{ cinder_ceph_client_uuid }}{%endraw%}"
       when: "{{ rpco_deploy_ceph | bool }}"
 
+    # We need to write out a specific log_hosts block so that we can set
+    # the required kernel parameters for the ELK components to startup.
```

Comment:
This is for production as well, I wasn't sure where to put it, so `conf.d` may be a better place,